editoast/src/schema/rolling_stock.rs

```diff
@@ -165,4 +259,37 @@ mod tests {
             StatusCode::NOT_FOUND
         );
     }
+
+    #[actix_test]
+    async fn get_light_rolling_stock() {
+        let manager = ConnectionManager::<PgConnection>::new(PostgresConfig::default().url());
+        let db_pool = Data::new(Pool::builder().max_size(1).build(manager).unwrap());
+
+        let mut rolling_stock_form: RollingStockForm =
+            serde_json::from_str(&include_str!("../tests/example_rolling_stock.json"))
```

Comment:
You should have a warning from clippy.

```suggestion
            serde_json::from_str(include_str!("../tests/example_rolling_stock.json"))
```